libs/application/templates/parental-leave/src/fields/InReviewSteps/index.tsx

```diff
@@ -141,61 +109,6 @@ const InReviewSteps: FC<FieldBaseProps> = ({ application, refetch }) => {
                 )}
             </Button>
           </Box>
-          <Box display="inlineBlock">
```

Comment:
Why all these codes have been removed?

Reply:
I took these out since Edit flow is not yet ready, and I didn't want to leave so much commented code hanging around. So I left it in a branch on my machine, and tried to make main a bit cleaner until the functionality is fully ready. What do you think?

Reply:
Okay, I understand, that's sound good, I would probably add a TODO comment and a link to the github branch so we know where it is and what happened

Alternatively, would be to create a `rfc.index.tsx` or something like this, and keep the review code there, up to you on this 👍

Reply:
Nice, I like the separate file idea (less branches to keep track off and then we can delete it when ready). I've added that file and a comment to `index.tsx`